tests/test_update_package.py

```diff
@@ -0,0 +1,2 @@
+# C1: Run "package update package". Expect example files are removed and
```

Comment:
the command will be `package update`, not `package update package`

Expect .git and wanted files are safe copied, example files are not created.

We would like C2, C3... where some files are present in the target but not overwritten, etc.  Checking all the expected behavior.